_reports/GuangyuanYang.md

```diff
@@ -164,7 +164,7 @@ investigation/review. Based on this feature, I implemented a
 [script](https://gist.github.com/yzgyyang/13ebb46a5c90ac624b5c6eb43deedc0d) to
 demonstrate the ability of bootstrapping coala Go bears on a bare-metal FreeBSD
 system
-([Screen record for the demo](https://asciinema.org/a/0poVbHE7lqehV4islFo6cptjb)),
+([Screen record for the demo]()),
```

Comment:
I don't think so this is required. This was reporting InvalidLink because we were using old coala. You should run latest coala and verify if it still throws InvalidLink.

Reply:
I just followed up on @kriti21's PR and I think she installed the latest coala and still the bear complained.